keps/sig-api-machinery/3716-admission-webhook-match-conditions/kep.yaml

```diff
@@ -35,13 +35,13 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.28"
+latest-milestone: "v1.30"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.27"
   beta: "v1.28"
-  stable: "TBD"
+  stable: "v1.30"
```

Comment:
Would you also update `keps/prod-readiness/sig-api-machinery/3716.yaml` and add a `stable:` stanza?